templates/main.html

```diff
@@ -81,7 +81,7 @@ <h5 class="summary">{{ game[0] }}</h5>
         </form>
     </div>
 
-    <form method="POST" action="/account">
+    <form method="GET" action="/account">
         <input type="submit" value="View Account" />
```

Comment:
DIdn't notice this earlier but good catch, since all were doing to going to an another page we need to GET the data not POST the data that isnt there in the first place